### PR TITLE
Add custom chip group counter

### PR DESCRIPTION
**Pull request by albinmedoc**
## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] I have followed the [contribution guidelines](https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md)
    - [x] This PR is for a custom-card or documentation change and therefore directed to the `main` branch.
    - [ ] This PR is for a official card or any other directly to the integration related change and therefore directed to the `release` branch.

### CODE REVIEW

File: custom_cards/custom_chip_group_counter/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+title: Custom Chip Group Counter
+hide:
+    - toc
+---
+
+<!-- markdownlint-disable MD046 -->
+
+# Custom Card "Group Counter"
+
+![example-image-light](../../docs/assets/img/custom_card_group_counter_chip_light.png)
```
Comment (basbruss) [inline custom_cards/custom_chip_group_counter/README.md:11]: Path doesn't exist

File: custom_cards/custom_chip_group_counter/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  Initial release
+</details>
+
+## Description
+
+This cards show a chip with custom text representing how many entities in a group have a specific state. Pressing on the chip will toggle the entity group, eg turn on/off all lights.
+
+## Variables
+
+| Variable                                    | Default                  | Required | Notes                                                                                              |
```
Comment (basbruss) [inline custom_cards/custom_chip_group_counter/README.md:32]: Table doesn't display as supposed to

**Comment by albinmedoc** [comment]
I have now fixed the issues.

**Comment by albinmedoc** [comment]
I can't find my custom chip on the [website](https://ui-lovelace-minimalist.github.io/UI/).
Should I make a new PR and add the documentation there as well, or is this something you will do?

**Pull request merged**